.github/pull_request_template.md

```diff
@@ -7,17 +7,13 @@
 Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.
 
 - [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
-- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
-- [ ] All existing and new tests are passing.
-- [ ] I updated/added relevant documentation (doc comments with `///`).
-- [ ] The Flame analyzer (`./scripts/analyze.sh`) does not report any problems on my PR.
-- [ ] I have added examples for new features in `doc/examples`.
-- [ ] I have formatted my code with `./scripts/format.sh`.
+- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors and are passing (See [Contributor Guide]).
+- [ ] I updated/added relevant documentation (doc comments with `///`) and updated/added examples in `doc/examples`.
+- [ ] I have formatted my code with `./scripts/format.sh` and the Flame analyzer (`./scripts/analyze.sh`) does not report any problems.
 - [ ] I read and followed the [Flutter Style Guide].
 - [ ] I have added a description of the change under `[next]` in `CHANGELOG.md`.
+- [ ] I removed the `Draft` status, by clicking on the `Ready for review` button in this PR.
```

Comment:
Is this last box really necessary? Since it doesn't go into draft by default, so the person doing the PR must understand how a draft works?

Reply:
I was contemplating it, but if we keep it in, then @erickzanardo cant keep asking us if a PR is ready because then DRAFT literally has a use hehehe :D

Reply:
I usually ask because I see a lot PRs marked as draft, which isn't in draft anymore because github UI for putting/removing something in Draft is quite confusing...

Reply:
Fiiiine 😄